clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -448,96 +442,27 @@ class TempFileHandlerRAII {
 /// designated name.
 ///
 /// To unbundle, we just copy the contents of the designated section.
-///
-/// The bundler produces object file in host target native format (e.g. ELF for
-/// Linux). The sections it creates are:
-///
-/// <OFFLOAD_BUNDLER_MAGIC_STR><target triple 1>
-/// |
-/// | binary data for the <target 1>'s bundle
-/// |
```

Comment:
I think this part is still valid even w/o the removed "oo" mode

Reply:
Added this part back as well as the next one.